### ISSUE TITLE
Slow window/UI repositioning

### ISSUE THREAD

**Issue by jacksongoode** [kraxarn/spotify-qt#199]
### System information
ABI: x86_64-little_endian-lp64
App version: v3.9 (387630e) - lib v0.7
Build type: Release
D-Bus support: No
Device: spotify-qt (librespot) (Speaker)
KCrash support: No
Kernel: darwin 21.6.0
Product: macOS 12.6
Qt version: 6.3.1

### Detailed information
Dragging the window around is sluggish, I also think this happens when resizing. Is this just inherent in QT apps? I could be wrong.

### What happens?
See video:

https://user-images.githubusercontent.com/54308792/201175042-7a965095-1e28-4c1d-bbc1-f0503521cc5e.mov

### What should happen?
See video.

**Comment by kraxarn** [comment]
Apologies for completely forgetting about this, but is this still an issue? 😅 
I'm not able to reproduce the issue on macOS 15.6, but you could try enabling "Interface > General > Use native window" and see if that helps.

**Comment by orkoden** [comment]
Yes, this is still an issue. Resizing the window, scrolling, and other UI interactions are extremely sluggish for me. 

Enabling "Interface > General > Use native window" helps a lot. Maybe this should be the default. 

macOS 26.1 (25B78), MacBook Air M2

**Comment by orkoden** [comment]
Seems to no longer be an issue for me.

**Settings**
- Interface > General > Use native window enabled
- Interface > Appearance > Dark Theme enabled

macOS 26.3 (25D125)
